src/orderbooks.md

```diff
@@ -0,0 +1,248 @@
+3. Check that Order book was created. Open `Chainstate` calls -> Choose `orderBook.orderBooks` -> Switch off include option switch -> Run the query. A deleted order book should be missing.
+
+### Updating an Order Book
+
+If you need to modify the parameters of an existing order book, this tutorial will demonstrate the steps to update its details. You will learn how to adjust the price step, volume step, or any other relevant parameters.
+
+<!-- TODO. Finish, when implemented -->
+
+### Changing Order Book Status
+
+This tutorial will explain how to change the status of an order book. You will understand the different statuses available and how to transition between them, such as from "Trade" to "PlaceAndCancel" or "OnlyCancel" to "Stop."
+
+<!-- TODO. Finish, when implemented -->
+
+### Placing a Limit Order
+
+In this tutorial, you will learn how to place a limit order on an order book. We will guide you through the process of setting the desired price and volume for your order and executing the trade.
+
+1. Choose `orderBool.placeLimitOrder`
+2. Fill the details:
+
+![](./assets/placeLimitOrder.png)
+
+_The query included the following parameters:_
+
+```
+Price: 1.1 DAI (1100000000000000000)
+Amount: 100 ETH (100000000000000000000)
+Side: Bid
+Lifespan: Current timestamp + 30 days
+```
+
+3. Check that Limit order was opened. Open `Chainstate` calls -> Choose `orderBook.orderBooks` -> Switch off include option switch -> Run the query.
+
+![](./assets/createdLimitOrderQuery.png)
+_The provided image showcases the query to check the result of the limit order being created._
+
+![](./assets/createdLimitOrderResult.png)
+_The image provided demonstrates the successful creation of the limit order._
+
+### Canceling a Limit Order
+
+If you need to cancel a previously placed limit order, this tutorial will demonstrate the steps to do so. You will understand how to locate your order and successfully cancel it.
+
+1. Choose `orderBool.cancelLimitOrder`
+2. Fill the details:
+
+<!-- TODO. Finish, when implemented -->
+
+3. Check that Limit order was cancelled. Open `Chainstate` calls -> Choose `orderBook.orderBooks` -> Switch off include option switch -> Run the query.
+
+<!-- TODO. Finish, when implemented -->
+
+_The image provided demonstrates the successful cancellation of the limit order._
+
+## References
+
+If you are proficient in RUST and Substrate, we invite you to explore our codebase dedicated to order books. By following this link https://github.com/sora-xor/sora2-network/tree/master/pallets/order-book, you will gain access to the source code, allowing you to delve deeper into the implementation details and functionalities of order books within our system.
+
+For more transparency on the development of order books, we recommend exploring our corresponding GitHub milestone - https://github.com/sora-xor/sora2-network/milestone/4. This will provide you with additional insights into the progress and advancements in order book implementation.
```

Comment:
```suggestion
For more transparency on the development of order books, we recommend exploring the corresponding [GitHub milestone](https://github.com/sora-xor/sora2-network/milestone/4). This will provide you with additional insights into the progress and advancements in order book implementation.
```